### DISCUSSION TITLE
Search capability of talks

### DISCUSSION THREAD

**Discussion by WengerK** [webmardi/webmardi.ch#49] [Ideas]
Hello @Yago 

We discuss an idea during our latest team minute and would like to know what do you think about it.
We would like to propose a "search" capabilities (frontend side) for all our talks #47 (on an other page or on both?).

What do you think ? Is that easy to implement or not at all ? What do you think about it ?

**Comment by Yago** [comment]
Sure!

Here are the options we have:
1. We type in the search field and it filters the event using their title
2. We are using the input to do a more advanced/fuzzy search on certain data attributes using [Fuse.js](https://fusejs.io/) or [Lunr](https://lunrjs.com/)

In both cases, it's not very hard to do 😉 

This might look like:

![Webmardi - Monthly events in western Switzerland](https://user-images.githubusercontent.com/2025924/143089630-44077d6f-b884-4550-80b8-4f53e42fc57c.png)

**Reply by WengerK** [reply]
I would upvote for solution N°1 "We type in the search field and it filters the event using their title"

**Comment by WengerK** [comment]
May we go further by creating an issue in order to follow progress on this task ?

**Reply by Yago** [reply]
I just [release](https://github.com/webmardi/webmardi.ch/releases/tag/v1.1.1) a first version of this idea 🎉

**Comment by WengerK** [comment]
Whooooo neet !! I love it <3 ! 

Will present asap (wednesday) to the team
